plugins/org.eclipse.n4js.transpiler/src/org/eclipse/n4js/transpiler/TranspilerBuilderBlocks.xtend

```diff
@@ -637,6 +695,12 @@ public class TranspilerBuilderBlocks
 		return result;
 	}
 
+	public static def PropertyAssignmentAnnotationList _PropAssAnnoList(Annotation[] annotations) {
```

Comment:
typo? `Prop_s_As` instead of `Ass`

Reply:
It was correct. `_Prop[erty]Ass[ignment]Anno[tation]List`.
But I remove the abbreviation to avoid confusion.